docs/content/mobile/android/setup.md

```diff
@@ -6,18 +6,18 @@
     * [Android Studio](https://developer.android.com/studio/) -  Once installed, open Android Studio. Ensure to install:
        * The Android SDK
        * The Android Studio emulator
-    * [Android NDK r13b](https://developer.android.com/ndk/downloads/older_releases) - Extract it to a directory of your choice. Note down this directory path as you will be needing it in the following steps.
-    * [JDK 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
-  * (Optional) [Unity Remote](https://play.google.com/store/apps/details?id=com.unity3d.genericremote) - this is Unity’s solution for faster development iteration times.
+    * [Java Platform, Standard Edition Development Kit (JDK)](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) - download and install the latest version.
+  1. (Optional) Download and unzip [Android NDK r16b](https://developer.android.com/ndk/downloads/older_releases), you only need this if you want to build for Android using [IL2CPP](https://docs.unity3d.com/Manual/IL2CPP.html). Extract it to a directory of your choice. Note down this directory path as you will be need to input the path into Unity later.
+  1. (Optional) [Unity Remote](https://play.google.com/store/apps/details?id=com.unity3d.genericremote) - this is Unity’s solution for faster development iteration times.
 
 ## Set up your Unity Editor
 After installing these dependencies, open the Unity project in `<path-to-your-project>/workers/unity`.
 
-  1. In the Unity Editor, go to **File** > **Build Settings**. Select **Android** and then click on **Switch Platform**.
-  1. In the Unity Editor, go to **Edit** > **Preferences**. In the **External Tools** window, add the file paths to the tools in the **Android** section:
+  1. In your Unity Editor, go to **File** > **Build Settings**. Select **Android** and then select **Switch Platform**.
+  1. In your Unity Editor, go to **Edit** > **Preferences** on Windows or **Unity** > **Preferences** on macOS. In the **External Tools** window, in the **Android** section, input the paths to the SDK, JDK and NDK. The easiest way to ensure that Unity reads the filepath correctly is to use the browse option:
```

Comment:
```suggestion
  1. Still in your Unity Editor, add the file paths Unity needs via the **External Tools** window. To do this;
  <br/> * on Windows, go to **Edit** > **Preferences** 
  <br/> * on MacOS, go to **Unity** > **Preferences**. 
  <br/> In the **Android** section of the **External Tools** window,  input the paths to the SDK, JDK and NDK. The easiest way to ensure that Unity reads the file path correctly is to use the browse option:
```
Slight rejig as this has a couple of OS-specific options. Sorry I didn't spot this before.